performance-tests/bench/worker/ForwardAction.cpp

```diff
@@ -108,7 +108,7 @@ bool ForwardAction::init(const ActionConfig& config, ActionReport& report,
   // hence 0 and 1 are not valid vector sizes and will not work
   data_queue_.resize(queue_size > 0 ? queue_size + 1 : 2);
 
-  handler_.reset(new MemFunHandler<ForwardAction>(&ForwardAction::do_writes, *this));
+  event_ = OpenDDS::DCPS::make_rch<MemFunEvent<ForwardAction> >(shared_from_this(), &ForwardAction::do_writes);
```

Comment:
```suggestion
  event_ = OpenDDS::DCPS::make_rch<MemFunEvent<ForwardAction>>(shared_from_this(), &ForwardAction::do_writes);
```
Don't need space with C++11?